elements/rh-blockquote/docs/10-style.md

```diff
@@ -1,16 +1,51 @@
-A default blockquote consists of three required elements, a quote icon, text, 
-and attribution. It can be left- or center-aligned on large and small screens. A 
-left-aligned blockquote can also feature additional elements, like large text 
-styles and components on top or to the right whereas a center-aligned blockquote 
-can't include anything else.
-
-{% example palette="light",
-           width=623,
-           class="centered",
-           alt="blockquote breakdown",
-           src="../blockquote-style-1.svg" %}
-
-Only the left-aligned blockquote can include these additional elements {.footnote.footnote-alignment}
+A blockquote is a combination of elements used to give visual prominence 
+to a quotation. By default, a blockquote includes a quote icon, quotation 
+text, and citation text <strong>at a minimum</strong>. A blockquote may 
+also include the following optional elements:
+ - Emphasis border
+ - Logo
+ - Title and heading text
+ - Interactive elements or layouts like a video or card
+
+
+![Anatomy image of a blockquote with numbered annotations]({{ '../blockquote-anatomy.png' | url }}) {style="margin-block:var(--rh-space-2xl);--inline-img-max-width:653px;"}
+1. Quote icon
+2. Quotation text
+3. Citation text
+4. Logo or text placeholder
+5. Emphasis border
+
+## Sizes
+
+![Image of two blockquotes, default size on the left and large size on the right]({{ '../blockquote-style-sizes.png' | url }}) {style="margin-block:var(--rh-space-2xl);--inline-img-max-width:872px;"}
+
+<table style="width:100%">
```

Comment:
prefer markdown tables where possible. here you can apply the `.full-width` class as in https://github.com/RedHat-UX/red-hat-design-system/blob/afd650cf1752384a70552518a968eb678d21e499/elements/rh-stat/docs/40-accessibility.md?plain=1#L10-L15